ddtrace/ext/git.py

```diff
@@ -65,6 +65,11 @@ def normalize_ref(name):
     return _RE_TAGS.sub("", _RE_ORIGIN.sub("", _RE_REFS.sub("", name))) if name is not None else None
 
 
+def is_ref_a_tag(ref):
+    # type: (Optional[str]) -> bool
+    return "tags/" in ref if ref else False
```

Comment:
Is it possible to have `tags/` in a ref and it not be a tag? eg., a contrived example might be if someone creates a branch called `kyle/tags/new-feature`. Or does git consider anything with `tags/` a tag?

Reply:
This will indeed give false positives. We could be a bit more strict but I don't think we can eliminate false positives 100%, so that's how we decided to do it.

Reply:
can/should we keep it as `"origin/tags" in ref or "refs/heads/tags" in ref` (as it was before)? Or we intentionally want to loosen it here?

Reply:
I changed that because I realized we had two different checks in place for the same thing. I left this one because it's the check we have in other tracers, even though it's a bit less strict I favoured consistency.